### ISSUE TITLE
Delay values of 0 prevent any delays from being applied with no warning.

### ISSUE THREAD

**Issue by Hammer-Inc** [SpectoLabs/hoverfly#887]
## Description of the bug
Adding any row that has a delay of 0 to globalActions.delays causes no other rows to be applied by Hoverfly
## Steps to reproduce the issue
```
hoverctl start webserver --listen-on-host 0.0.0.0
hoverctl import hoverfly.json
hoverctl mode simulate
```

### hoverfly.json
```
{
  "data": {
    "pairs": [
       {
        "request": {
          "path": [
            {
              "matcher": "exact",
              "value": "/delay/1"
            }
          ],
          "method": [
            {
              "matcher": "exact",
              "value": "GET"
            }
          ],
          "destination": [
            {
              "matcher": "exact",
              "value": "whocares.org"
            }
          ],
          "scheme": [
            {
              "matcher": "exact",
              "value": "http"
            }
          ]
        },
        "response": {
          "status": 200,
          "body": "Success"
        }
      },
       {
        "request": {
          "path": [
            {
              "matcher": "exact",
              "value": "/delay/0"
            }
          ],
          "method": [
            {
              "matcher": "exact",
              "value": "GET"
            }
          ],
          "destination": [
            {
              "matcher": "exact",
              "value": "whocares.org"
            }
          ],
          "scheme": [
            {
              "matcher": "exact",
              "value": "http"
            }
          ]
        },
        "response": {
          "status": 200,
          "body": "Success (No Delay)"
        }
      }
    ],
    "globalActions": {
      "delays": [{
          "urlPattern": "./delay/1",
          "httpMethod": "GET",
          "delay": 1000
        },{
          "urlPattern": "./delay/0",
          "httpMethod": "GET",
          "delay": 0
        }],
      "delaysLogNormal": []
    }
  },
  "meta": {
    "schemaVersion": "v5",
    "hoverflyVersion": "v1.1.3"
  }
}
```
### Observed result
**Hoverfly error messages seen (If none, say none)**
None
**If possible, add screenshots to help explain your problem**

### Expected result
Hitting the simulated /delay/1 url should result in a 1 second delay but no such delay occurs, Removing or modifying the /delay/0 row results in the expected outcome.
```
curl -X GET \
  http://localhost:8500/delay/1 \
  -H 'Accept: */*' \
  -H 'Accept-Encoding: gzip, deflate' \
  -H 'Cache-Control: no-cache' \
  -H 'Connection: keep-alive' \
  -H 'Host: localhost:8500' \
  -H 'cache-control: no-cache'
```

## Additional relevant information
1. Hoverfly version: v1.1.3
2. Anything that might help us to diagnose the problem

**Comment by tommysitu** [comment]
@Hammer-Inc thank you for raising the issue, the fix will be in the next release :)

**Comment by tommysitu** [comment]
fixed in v1.1.4
